PersonBook/PersonBook/PersonBook/Service/PersonService.cs

```diff
@@ -1,24 +1,33 @@
-﻿using PersonBook.Entity;
+﻿
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 
 namespace PersonBook.Service
 {
-    class PersonService : IPersonService<Person>
+    using Entity;
+
+     class PersonService : IPersonService<Person>
```

Comment:
`IPersonService<Person>` doesn't need to be generic. it can be just `IPersonService`

Reply:
Done